index.html

```diff
@@ -33,5 +33,183 @@ <h2 class="social">LET'S CONNECT</h2>
                 </ul>    
             </div>     
         </section>
+
+        <section class="works">
+        
+            <div class="container">
+                <div class="card">
+                    <img src="./img/Snapshoot Portfolio.png" alt="snapshot poortfolio" class="snapshot">
+                    <div class="works-description">
+                        <div class="works-intro">
+                            <h1 class="project-title">Tonic</h1>
```

Comment:
- [ ] Kindly replace this `h1` tag with another `hx` tag because there should be only one `h1` tag in the web page which is the main heading of the page (this applies to all the other `h1` tags that follow this one) 👍